pom.xml

```diff
@@ -308,7 +291,7 @@
                         </goals>
                         <phase>package</phase>
                         <configuration>
-                            <repository>${project.artifactId}</repository>
+                            <repository>eu.gcr.io/census-rm-ci/rm/${project.artifactId}</repository>
```

Comment:
The step of the travis build which tries to retag the built image name can be deleted with this change.